File: tests/csharp/csharp_operators/unary_minus.cs
// vybe-test: csharp/csharp_operators/unary_minus
// origin: languages/csharp/tests/csharp/test_csharp_operators.rs

void __Check(string got, string want) {
    if (got != want) {
        Console.WriteLine("FAIL: want [" + want + "] got [" + got + "]");
        throw new Exception("assertion failed");
    }
}

int x = 42;
__Check((-x).ToString(), "-42");

File: tests/fortran/arithmetic/unary_minus.f90
! vybe-test: fortran/arithmetic/unary_minus
! origin: languages/fortran/tests/fortran/test_arithmetic.rs
program t
integer :: x = 5
if ((-x) /= -5) then
    print *, "FAIL: want [-5] got [", -x, "]"
    stop 1
end if
end program t
